shared/templates/template_BASH_audit_rules_dac_modification

```diff
@@ -11,8 +11,15 @@ for ARCH in "${RULE_ARCHS[@]}"
 do
 	PATTERN="-a always,exit -F arch=$ARCH -S %ATTR%.*"
 	GROUP="perm_mod"
+{{% if product != "rhel6" %}}
 	FULL_RULE="-a always,exit -F arch=$ARCH -S %ATTR% -F auid>=1000 -F auid!=4294967295 -F key=perm_mod"
+{{% else %}}
+	FULL_RULE="-a always,exit -F arch=$ARCH -S %ATTR% -F auid>=500 -F auid!=4294967295 -k perm_mod"
```

Comment:
aren't `-F key=value` and -k value` equivalent? or is it a legit difference between rhel6 and 7?

Assuming they are the same, could you please instead set a variable for the auid and just do a condition around that?